pkg/operator/ceph/cluster/osd/remove.go

```diff
@@ -40,18 +42,29 @@ func removeOSD(context *clusterd.Context, namespace, deploymentName string, id i
 
 	// first reweight the OSD to be 0.0, which will begin the data migration
 	o, err := client.CrushReweight(context, namespace, id, 0.0)
+	already_purged := false
```

Comment:
nit: i'd suggest `alreadyPurged`. Underscores are not generally used in golang.

Reply:
done